### PR TITLE
PLAT-441 Remove ssh and telnet dependencies from org.crashub:jcr.core

### PR DESCRIPTION
**Pull request by ilx**

### CODE REVIEW

File: core/pom.xml [hunk 1]
```diff
@@ -2,12 +2,12 @@
   <parent>
     <artifactId>jcr.parent</artifactId>
     <groupId>org.crashub</groupId>
-    <version>1.0.1-SNAPSHOT</version>
+    <version>1.1.0-SNAPSHOT</version>
```
Comment (hierynomus) [inline core/pom.xml:5]: Let's give it a slightly less officially looking version number!

Comment (hierynomus) [inline core/pom.xml:5]: Something like 1.0.1-xebialabs-SNAPSHOT

Comment (ilx) [reply core/pom.xml:5]: xebialabs is classifier which we will use - that way you cannot pull crashub version

Comment (hierynomus) [reply core/pom.xml:5]: OK, missed that.

**Pull request merged**